com.unity.mobile.notifications/Tests/Runtime/Android/AndroidNotificationSendingTests.cs

```diff
@@ -184,7 +184,8 @@ public IEnumerator NotificationIsScheduled_NotificationStatusIsCorrectlyReported
         var status = AndroidNotificationCenter.CheckScheduledNotificationStatus(originalId);
         Assert.AreEqual(NotificationStatus.Scheduled, status);
 
-        yield return WaitForNotification(8.0f);
+        yield return WaitForNotification(120.0f);
```

Comment:
It only waits until notification is delivered, so 120s only if it is not delivered. Testing the repeating notification showed, that Androids alam with inexact repeated scheduling roughly operates with reliability of minutes, so I'm switching all cases where we wait for scheduled notification to 120s because that's the reliable figure I came up with.
I guess the 8 seconds was near 100% fine and test instability was actually caused by the lack of the later wait. However, unstable test means I rerun the entire suite, which takes an order of magnitude longer, so extra second here or there is worth paying.
As for warning, we later check if notification status is delivered. It won't be, unless notification actually comes in between this wait and later query, but with such high timeout it almost certainly wont happen.